File: examples/ruby/spec/selenium_manager/usage.rb
require 'selenium-webdriver'

def setup_without_selenium_manager
  service = Selenium::WebDriver::Chrome::Service.new(path: '/path/to/chromedriver')
  driver = Selenium::WebDriver.for(:chrome, service: service)
  driver.get("https://www.selenium.dev/documentation/selenium_manager/")
  driver.quit
end

def setup_with_selenium_manager
  driver = Selenium::WebDriver.for(:chrome) # Selenium Manager handles the driver automatically
  driver.get("https://www.selenium.dev/documentation/selenium_manager/")
  driver.quit
end

File: examples/python/tests/selenium_manager/usage.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.chrome.service import Service

def setup_without_selenium_manager():
    chrome_service = Service(executable_path='path/to/chrome.exe')
    driver = webdriver.Chrome(chrome_service)
    return driver
    
def setup_with_selenium_manager():
    driver = webdriver.Chrome()
    return driver